docs/guides/writingafilter.md

```diff
@@ -1,19 +1,31 @@
-# Writing a Filter
+# Writing an Event Filter
 
-Filters are an important part of TriggerMesh's event routing mechanism.
-Content-based event filtering is expressed in Google's
-[Common Expression Language](https://opensource.google/projects/cel).
+Filters are an important part of TriggerMesh's event routing mechanism. They allow for filtering events based on the content of the payload. This content-based event filtering is expressed with Google's
+[Common Expression Language](https://opensource.google/projects/cel) within the TriggerMesh `Filter` API specification.
 
-## Prerequisites
+!!! Info "Prerequisites"
+    You need a working TriggerMesh platform installation. See the [installation steps](installation.md).
+    You can verify that the API is available with the following command:
 
-1. K8s cluster and configured kubectl
-1. [Knative Serving and Eventing Operators](https://knative.dev/docs/admin/install/knative-with-operators/)
-1. [TriggerMesh Function](https://github.com/triggermesh/function)
+    ```console
+    $ kubectl get crd|grep filters.routing
+      filters.routing.triggermesh.io                       2021-10-06T09:01:33Z
+    ```
```

Comment:
grep is totally unnecessary here.

```suggestion
    ```console
    $ kubectl get crd filters.routing.triggermesh.io
    NAME                             CREATED AT
    filters.routing.triggermesh.io   2021-10-06T09:01:33Z
    ```
```